UI Updated

gstewar7 opened the pull request:
A work in progress, but UI is being updated with the new font I added. Inventory coming very soon.

gibarr28 committed:
UI Updated

A work in progress, but UI is being updated with the new font I added. Inventory coming very soon.

gibarr28 committed:
UI Completely Updated

All of the UI is updated. Just need the inventory and end credits theme, and that should be it. Let me know your other plans so I can add them to my list and draw them for the inventory.

gibarr28 committed:
Recommit with controlIDs being idfferent in layouts

Going to recommit this to see if the assets will actually appear. I noticed that the Control ID will be changed based on the results

gibarr28 committed:
InventoryUI

InventoryUI should show up since I toggle it visible, which means you'll see it as soon as you start the game.

gibarr28 commented:
Seems there are conflicting files between Pestilence.unity and default-2021.dwlt.

gibarr28 committed:
Day 1 Patch

The day 1 patch is here. Includes qol stuff like a new inventory/pause menu, and an ending with a new ending song.
